src/plugins/git/loadRepositoryTest.sh

```diff
@@ -23,8 +23,12 @@ usage() {
 
 fetch() {
   tmpdir="$(mktemp -d)"
-  node "${SOURCECRED_BIN:-./bin}/createExampleRepo.js" "${tmpdir}"
-  node "${SOURCECRED_BIN:-./bin}/loadAndPrintGitRepository.js" "${tmpdir}"
+  SOURCECRED_DIRECTORY="${tmpdir}" \
+    node "${SOURCECRED_BIN:-./bin}/sourcecred.js" \
+    load --plugin git \
+    sourcecred/example-git
+  cat "${tmpdir}"/data/sourcecred/example-git/git/repository.json \
```

Comment:
This is a “useless use of `cat(1)`”. You can simply:

```shell
<"${tmpdir}"/data/sourcecred/example-git/git/repository.json \
  python -m json.tool
```

(The redirection can also go after the command, if you prefer.)

It looks like `python -m json.tool` also accepts a filename as a
command-line argument? If this is the case, then it’s definitely better
to use that form. Tools can be orders of magnitude faster with this
formulation, because they get a file as opposed to an arbitrary input
stream, which grants them more fine-grained control over I/O. (I’ve
noticed this especially with `jq(1)`.)